Database/Corrections/QuestieQuestFixes.lua

```diff
@@ -230,6 +231,17 @@ function QuestieQuestFixes:Load()
             [QuestieDB.questKeys.sourceItemId] = 18746,
             [QuestieDB.questKeys.zoneOrSort] = 1637,
         },
+        [7669] = { --#1449
+            [QuestieDB.questKeys.name] = "Again Into the Great Ossuary",
+            [QuestieDB.questKeys.startedBy] = {{928,},nil,nil,},            -- Quest is started by Lord Grayson Shadowbreaker
+            [QuestieDB.questKeys.finishedBy] = {{928,},nil,nil,},           --       & ended*
+            [QuestieDB.questKeys.requiredLevel] = 58,
+            [QuestieDB.questKeys.questLevel] = 60,
+            [QuestieDB.questKeys.requiredRaces] = 178,                      -- Any race can take on quest
+            [QuestieDB.questKeys.requiredClasses] = 64,                     -- This quest is for the Shaman class
+            [QuestieDB.questKeys.zoneOrSort] = -141,                        -- <0: QuestSort.dbc ID
+            [QuestieDB.questKeys.specialFlags] = 1,                         -- quest is repeatable
```

Comment:
Is this quest only a repeatable quest or is it a sub quest of some other quest? If it is, then you'll need to add a parent-child relation as well. Otherwise the quest will never be marked as complete and always show on the map.

Reply:
On the wowhead page it just says it's repeatable. I didn't see anything about it having parent or child quests.

Reply:
Although, on the [Paladin version](https://classic.wowhead.com/quest=7666/again-into-the-great-ossuary) it does have a "required" quest. So, maybe wowhead is just missing some information.

Reply:
Most likely, yes. Can't really tell how to get more information on this without testing it ingame.